apps/components-e2e/src/components/expandable-section/expandable-section.e2e.ts

```diff
@@ -25,31 +25,31 @@ fixture('Expandable Section').page('http://localhost:4200/expandable-section');
 
 test('should toggle', async (testController: TestController) => {
   await testController.click(toggle);
-  await testController.expect(await open.textContent).eql('1');
+  await testController.expect(await open.textContent).eql('true');
   await testController.click(toggle);
-  await testController.expect(await open.textContent).eql('0');
+  await testController.expect(await open.textContent).eql('false');
 });
 
 test('should execute click handlers when not disabled', async (testController: TestController) => {
-  await testController.expect(await open.textContent).eql('0');
+  await testController.expect(await open.textContent).eql('false');
   await testController.click(buttonOpen);
-  await testController.expect(await open.textContent).eql('1');
+  await testController.expect(await open.textContent).eql('true');
 });
 
 test('should not execute click handlers when disabled', async (testController: TestController) => {
   await testController.click(buttonDisable);
   await testController.click(buttonOpen);
-  await testController.expect(await open.textContent).eql('0');
+  await testController.expect(await open.textContent).eql('false');
 });
 
 test('should close after disabling open section', async (testController: TestController) => {
   await testController.click(buttonOpen);
-  await testController.expect(await open.textContent).eql('1');
+  await testController.expect(await open.textContent).eql('true');
   await testController.click(buttonDisable);
-  await testController.expect(await open.textContent).eql('0');
+  await testController.expect(await open.textContent).eql('false');
 });
 
 test('should open on label click', async (testController: TestController) => {
   await testController.click(labelHeader);
-  await testController.expect(await open.textContent).eql('1');
+  await testController.expect(await open.textContent).eql('true');
```

Comment:
BIT: could be chained to improve speed and flakyness.

Reply:
Good point will do 💪